android/app/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,8 @@
         android:name=".MainActivity"
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
-        android:windowSoftInputMode="adjustResize">
+        android:windowSoftInputMode="adjustResize"
+        android:launchMode="singleTask">
```

Comment:
Remove, this causes other issues

Reply:
Did you test without having a default app to attach images/videos?

Reply:
Although before it was set to singleInstance, what is the difference?

Reply:
Seems like without `singleTask`, we get this situation (Mattermost gets launched within Firefox app context):

<img width="200" alt="Mattermost" src="https://user-images.githubusercontent.com/887849/72894500-35c7d280-3cfa-11ea-9f34-45588cc5c7f9.jpg"><img width="200" alt="Firefox" src="https://user-images.githubusercontent.com/887849/72894501-35c7d280-3cfa-11ea-8fd5-03e80da7c804.jpg">